- fix: record screen on Iphone

BrianTran24 opened the pull request:
#46 : Fix issue  : Invalid value: Not in inclusive range

tranhieu1998-it committed:
- fix: record screen on Iphone

ueman commented:
Please include the percentage updates in another PR. That makes it easier to review individual changes, and roll them back if needed.

tranhieu1998-it committed:
style: format code

BrianTran24 commented:
> Please include the percentage updates in another PR. That makes it easier to review individual changes, and roll them back if needed.

Thanks for your review. This was my mistake. Percentance feature is not tested, And it is not really necessary to create PR right now.
Currently, I have removed this feature in PR, and adjusted the code format to common format.

tranhieu1998-it committed:
style: format style

martijn00 committed:
Merge branch 'main' into main
